linkerd/docs/protocol-h2.md

```diff
@@ -71,6 +71,11 @@ When TLS is not configured, h2 servers accept both
 requests.  Plaintext clients are currently only capable of issuing
 prior-knowledge requests.
 
+
+<aside class="warning">
+Note that HTTP/2 clients should be careful not to send any DATA frame whose body is longer than the current window size minus the size of the response classification buffer (8 kb), as doing so may cause the stream to hang.
```

Comment:
The parameter is called `retryBufferSize`: https://linkerd.io/config/1.3.1/linkerd/index.html#http-2-service-parameters and if configurable.  8kb is just the default.

Also we need to update the docs at https://linkerd.io/config/1.3.1/linkerd/index.html#http-2-service-parameters with the new default value.

Reply:
Oh, okay, I didn't realise this was configurable; I looked at the config classes, but I was looking in the wrong place (`H2ClassifierConfig`). I'll update the docs. Do you think this warning should go next to where we describe those config fields instead of where it is at the moment?